suite-common/wallet-utils/src/__fixtures__/antiFraud.ts

```diff
@@ -127,6 +127,30 @@ export const getIsFakeTokenPhishingFixtures = [
         } as TokenDefinitions,
         result: false,
     },
+    {
+        testName: 'one fake, one zero-value legit token tx',
+        transaction: {
+            symbol: 'matic',
+            amount: '0',
+            tokens: [
+                { standard: 'ERC20', contract: '0xA', anmount: '1' },
+                { standard: 'ERC20', contract: '0xB', amount: '0' },
+            ],
+        } as WalletAccountTransaction,
+        tokenDefinitions: {
+            coin: {
+                error: false,
+                isLoading: false,
+                data: ['0xb'],
+            },
+            nft: {
+                error: false,
+                isLoading: false,
+                data: ['0xn', '0xf'],
+            },
+        } as TokenDefinitions,
+        result: true,
+    },
```

Comment:
can you also provide other scenarios, such as:
- only zero legit tokens
- only zero non legit tokens
- legit balance 1, non-legit balance 0

so that it is covered and we do not need to return to it

maybe also one case with nft so we know that it does not break anything in case of non-ERC20

Reply:
b590ae49034370ce7cdd33d150182c26138bd139